underlay/src/main/java/bio/terra/tanagra/query/bigquery/BQQueryRunner.java

```diff
@@ -504,7 +516,7 @@ public HintQueryResult run(HintQueryRequest hintQueryRequest) {
                           sqlRowResult.get(attributeColName, DataType.STRING).getStringVal());
               if (attribute.isValueDisplay()
                   || attribute.getRuntimeDataType().equals(DataType.STRING)) {
-                // This is one value/count pair of an enum values hint.
+                // This is one value/count a pair of an enum values hint.
```

Comment:
I don't follow this change. Can you explain?

Reply:
Just a grammar warning fixed - added the article 'a', no changes to existing comment

Reply:
Yes, it's a minor thing, but I don't think this edit makes sense. Maybe changing it to "(value, count) pair" would be better wording?

Reply:
Oh, apologies got it. I didnt grasp the entire context of the comment. My bad. Reverting it

Reply:
Updating it to reflect key,value pair

Reply:
Thank you!